docs/topics/rotate-certs.md

```diff
@@ -0,0 +1,127 @@
+
+1. `aks-engine rotate-certs` does **not** guarantees backwards compatibility. If you deployed with `aks-engine` version `0.27.x`, you should prefer executing the certificate rotation process with version `0.27.x`.
+
+### Parameters
+
+|Parameter|Required|Description|
+|-----------------|---|---|
+|--api-model|yes|Relative path to the API model (cluster definition) that declares the expected cluster configuration.|
+|--location|yes|Azure location where the cluster is deployed.|
+|--ssh-host|yes|FQDN, or IP address, of an SSH listener that can reach all nodes in the cluster.|
+|--linux-ssh-private-key|yes|Path to a valid private SSH key to access the cluster's Linux nodes.|
+|--certificate-profile|no|Relative path to a JSON file containing the new set of certificates.|
+|--generate-new-certificates|no|Delegate generation of the new set of certificates, required if --certificate-profile is not specified.|
+|--continue|no|Resume a previous unsuccessful execution.|
+|--abort|no|Clean temporary artifacts left behind after an unsuccessful execution.|
+
+### Simple steps to rotate certificates
+
+Once you have read all the [requirements](#pre-requirements), run `aks-engine rotate-certs` with the appropriate arguments:
+
+```bash
+./bin/aks-engine rotate-certs \
+  --location <resource-group-location> \
+  --api-model <generated-apimodel.json> \
+  --linux-ssh-private-key <private-SSH-key> \
+  --ssh-host <apiserver-URI> \
+  --generate-new-certificates
+```
+
+For example,
+
+```bash
+./bin/aks-engine rotate-certs \
+  --location "westus2" \
+  --api-model "_output/my-cluster/apimodel.json" \
+  --linux-ssh-private-key "~/.ssh/id_rsa" \
+  --ssh-host "my-cluster.westus2.cloudapp.azure.com" \
+  --generate-new-certificates
+```
+
+> Fetching a new set of certificates from Key Vault is not supported at this point.
+
+## Under The Hood
+
+A Kubernetes cluster relies on multiple PKIs to secure the communication between its components (apiserver, kubelet, etcd, etc). On an AKS Engine cluster, these multiple PKIs share a single certificate authoritiy (CA). In order to reduce downtime, `aks-engine rotate-certs` rotates each of these PKIs (etcd, service account tokens, apiserver-kubelet) in sequence.
+
+Each of these are idempotent discrete steps. The cluster is expected to reach a healthy state after each step is executed. If the certificate rotation process halts before completion due to a transient issue (e.x.: network connectivity), rerun `aks-engine rotate-certs` using the `--continue` flag so the process can be resumed at the point where it failed.
```

Comment:
I assume transient network issues will *very rarely* produce error outcomes, because all of the underlying network-dependent operations will retry aggressively, right?

Reply:
right